src/panel/pages/mismatch/Mismatch.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ComponentProps } from "react";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBomb } from "@fortawesome/free-solid-svg-icons";
+import { useDevtoolsContext } from "../../context";
+
+export const Mismatch: FC<ComponentProps<typeof Container>> = (props) => {
+  const { version } = useDevtoolsContext();
+  return (
+    <Container {...props}>
+      <Icon icon={faBomb} />
+      <Header>Version Mismatch</Header>
+      <Hint>
+        Expected devtools exchange (@urql/devtools) version{" "}
```

Comment:
Can we make this a little more user-friendly with some upgrade instructions? :)

Reply:
I thought this would be pretty obvious to anyone with the addition of `(@urql/devtools)` - what were you thinking?

Reply:
Done